test/src/unit-cppapi-updates.cc

```diff
@@ -186,7 +186,12 @@ TEST_CASE(
 
   // Submit query
   rc = test::submit_query_wrapper(
-      ctx, array_name, &query_w2, server_buffers_, serialized);
+      ctx,
+      array_name,
+      &query_w2,
+      server_buffers_,
+      serialized,
+      layout != TILEDB_UNORDERED);
```

Comment:
Can you explain why you needed to do that? I guess your changes in `finalize()` was making that (unrelated?) test fail but I don't understand why.

Reply:
Fixed now... It was a test writing empty buffers, which didn't update the written buffer list.